VirtoCommerce.Platform.Core/DynamicProperties/DynamicProperty.cs

```diff
@@ -31,6 +31,7 @@ public class DynamicProperty : AuditableEntity, ICloneable
         /// <summary>
         /// The storage property type 
         /// </summary>
+        [JsonConverter(typeof(StringEnumConverter))]
```

Comment:
This attributed is redundant. This behaviour can be achieved on global application level via ```  jsonFormatter.SerializerSettings.Converters.Add(new StringEnumConverter());```